.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish to testpypi repo
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to TestPyPI
```

Comment:
Does this publish to TestPyPi, or PyPi?

Reply:
It publishes to PyPi, I had tested it using TestPyPI, then changed it so it publishes to PyPi but forgot to change the name of the action

Reply:
I will do so now